src/nfts/NftImage.tsx

```diff
@@ -71,7 +68,7 @@ export default function NftImage({
       sendImageLoadEvent('No nft metadata')
       setStatus('error')
     }
-  }, [nft])
+  }, [`${nft.contractAddress}-${nft.tokenId}`])
```

Comment:
Could we do this instead?
```suggestion
  }, [imageUrl])
```

And use `imageUrl` in the condition inside?

Reply:
Weirdly enough image url isn't unique. Ipermatch for example has multiple NFTs with the same image url.